src/inausoft.netCLI/OptionAttribute.cs

```diff
@@ -26,16 +31,33 @@ public class OptionAttribute : Attribute
         /// <summary>
         /// Initializes a new instance of <see cref="OptionAttribute"/> with specified name and description.
         /// </summary>
-        /// <param name="name"></param>
+        /// <param name="naming"></param>
         /// <param name="helpDescription"></param>
-        public OptionAttribute(string name, string helpDescription = "")
+        public OptionAttribute(string naming, string helpDescription = "")
         {
-            if (string.IsNullOrWhiteSpace(name) || name.StartsWith("-"))
+            if (naming == null)
             {
-                throw new ArgumentException(nameof(name));
+                throw new ArgumentNullException(nameof(naming));
             }
 
-            Name = name;
+            if (naming.Contains("-"))
+            {
+                throw new ArgumentException($"{nameof(naming)} cannot include '-'.");
+            }
+
+            var names = naming.Split('|');
```

Comment:
I agree we can go with explicit values, but it would require passing '-' as well - so it would need to be something like this:
`
[Option("--poption", ShortName = "-p")]
		public string Poption { get; set; }
`
But let's leave it for version 3 - as it will be braking change.